modules/cli/src/main/scala/scala/cli/exportCmd/SbtProjectDescriptor.scala

```diff
@@ -293,11 +299,15 @@ final case class SbtProjectDescriptor(
               val sep = "%%"
               (sep, suffixOpt0)
           }
-          val scope0 = if (scope == Scope.Test) "% Test" else ""
+          val scope0 =
+            // FIXME This ignores the isCompileOnly when scope == Scope.Test
+            if (scope == Scope.Test) "% Test"
```

Comment:
Do we have `TestProvided` scope? Is this even possible? MAybe there shouldn't be an option to define them in test scope?

Reply:
I've looked into it and I didn't find solution how to define compile-only dependencies only for test scope in SBT. 

It might be sufficient to define this dependency as main scope, which would enable exported project to work.